lib/spans/span-event.js

```diff
@@ -259,17 +265,18 @@ class DatastoreSpanEvent extends SpanEvent {
       attributes.database_name = null
     }
 
-    if (attributes.host) {
-      this.addAttribute('peer.hostname', attributes.host)
-      this.addAttribute('server.address', attributes.host)
+    // `attributes.host` was translated in SpanEvent class
+    // use `server.address` to assign to `peer.hostname`
+    const serverAddress = attributes[SERVER_ADDRESS]
```

Comment:
I'm not sure I follow what this comment is trying to convey.

Reply:
it's prob not needed. i was commenting on why this is done the way it is now compared to before